appservice/src/main/java/nu/yona/server/device/rest/DeviceController.java

```diff
@@ -350,9 +385,35 @@ public static ControllerLinkBuilder getRegisterDeviceLinkBuilder(UUID userId)
 
 	public static class DeviceResource extends Resource<DeviceBaseDto>
 	{
-		public DeviceResource(DeviceBaseDto device)
+		private static String sslRootCertificateCn;
+		private final boolean isRequestingDevice;
+
+		public DeviceResource(DeviceBaseDto device, boolean isRequestingDevice)
 		{
 			super(device);
+			this.isRequestingDevice = isRequestingDevice;
+		}
+
+		public static void setSslRootCertificateCn(String sslRootCertificateCn)
+		{
+			DeviceResource.sslRootCertificateCn = sslRootCertificateCn;
+		}
+
+		public boolean isRequestingDevice()
+		{
+			return isRequestingDevice;
+		}
+
+		@JsonProperty("sslRootCertCN")
+		@JsonInclude(Include.NON_EMPTY)
+		public Optional<String> getSslRootCertCn()
```

Comment:
❓  This is not different per device too, is it?

Reply:
See previous comment.